src/main/java/ru/hogwarts/repository/StudentRepository.java

```diff
@@ -0,0 +1,13 @@
+package ru.hogwarts.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import ru.hogwarts.entity.Student;
+
+import java.util.List;
+
+public interface StudentRepository extends JpaRepository<Student, Long> {
+    List<Student> findAllByAge(int age);
+    List<Student> findByAgeBetween (int maxAge, int minAge);
+
+    List<Student> findAllByFaculty_ID(long facultyId);
```

Comment:
```suggestion
    List<Student> findAllByFaculty_Id(long facultyId);
```
в факультете поле называется id